config/generator-theia/README.md

```diff
@@ -1,7 +1,7 @@
 # Theia Generator
 
 ## Getting started
-
+- `npm install` in theia/config/generator-theia
```

Comment:
could you rephrase: `npm install` to build

README already belongs to `theia/config/generator-theia`